### PR TITLE
Alternate throttling scope. Fix: #10

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- defers throttling scope to future implementations
- explains in FAQ why URL is the only interoperable choice.

### CODE REVIEW

**Review by unleashed — COMMENTED** [review]
@ioggstream I wanted to avoid the whole `acme-` prefix trap. The problem I find with it is that it opens the whole namespace of parameter names to everyone, and extensibility can become next to impossible if vendors pollute this namespace. _Even_ if we support multiple parameters from multiple vendors at once, I'd suggest we agree on a namespaced prefix for such parameters, ie. `x-acme-`, `x-scope-acme-`, etc - anything that can clearly distinguish registered parameters from vendor ones should be fine.

**Comment by richsalz** [comment]
Convention would be for the document to say that `X-` as a prefix is left for private use and experimentation.

**Comment by ioggstream** [comment]
@unleashed I pick `acme-` because of https://datatracker.ietf.org/doc/html/rfc6648 deprecating x- headers, 

* `x-acme`  is ok.
* just `x-` is not ok since private stuff becomes public soon (x-ratelimit-limit once was as a private experiment, you know ;) @richsalz

**Comment by unleashed** [comment]
@ioggstream so you mean mandating that such parameters be prefixed with `x-<vendor>-<parametername>`? If so, that would be fine with me.

**Comment by ioggstream** [comment]
I think we could recommend, not mandate.

Let's wait for @richsalz feedback on that option

**Comment by richsalz** [comment]
I don't have a preference between `x-...` and `x-microsoft-...` (for example)  The example `acme` above confuses me because of the ACME working group. But saying SHOULD follow the naming convention seems fine with me.  Others on the WG, more expert, will have opinions I'm sure. It's probably worth calling this item out specifically once you merge it.

**Comment by ioggstream** [comment]
@unleashed PTAL

**Review by unleashed — APPROVED** [review]
Looks good!

**Pull request merged**
